assert/assertions.go

```diff
@@ -6,19 +6,20 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/sergi/go-diff/diffmatchpatch"
```

Comment:
This import should not be in the block with stdlib imports.

Reply:
curious if we have explored `go-cmp` package for diffs.

Reply:
@peymanmortazavi I haven't.

Reply:
@peymanmortazavi I considered using `go-cmp` initally. Unfortunately `go-cmp` cannot produce structured diff so we can't customize them. This is why I decided to use diffmatchpatch here